packages/venia-ui/lib/RootComponents/Search/search.js

```diff
@@ -37,8 +37,8 @@ const Search = props => {
         queryParameter: 'category'
     });
     const queryVariable = categoryId
-        ? { inputText, categoryId }
```

Comment:
Ah, this does need to be `inputText` though since it is the variable name in the gql query.